src/test/java/net/sf/jabref/logic/importer/fetcher/MrDLibFetcherTest.java

```diff
@@ -0,0 +1,40 @@
+package net.sf.jabref.logic.importer.fetcher;
+
+import java.util.List;
+
+import net.sf.jabref.logic.importer.FetcherException;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.FieldName;
+import net.sf.jabref.testutils.category.FetcherTests;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import static org.junit.Assert.assertEquals;
+
+@Category(FetcherTests.class)
+public class MrDLibFetcherTest {
+
+    private MrDLibFetcher fetcher;
+    private BibEntry bibEntry;
+
+    @Before
+    public void setUp() {
+        fetcher = new MrDLibFetcher("", "");
+        bibEntry = new BibEntry();
+        bibEntry.setField(FieldName.TITLE, "lernen");
+    }
+
+    @Test
+    public void testPerformSearch() throws FetcherException {
+        List<BibEntry> bibEntrys = fetcher.performSearch(bibEntry);
+        assertEquals(false, bibEntrys.isEmpty());
```

Comment:
> Because the Servers response will be different every time.

Really? I thought that you create a concrete BibEntry for a valid publication and then check that a certain other publication (e.g. by the same author about a related topic) is returned as recommendation. But if your recommendation algortithm is completely unpredictable/random, then this approach does not work. 
The idea is also to have a proper test that shows whether the SSL problem is fixed or not (i.e. if at least the ci server can access the recommendation without SSL error)